langgraph/src/pregel/algo.ts

```diff
@@ -417,17 +418,14 @@ export function _prepareNextTasks<
         });
       }
     } else {
-      taskDescriptions.push({
-        name: packet.node,
-        input: packet.args,
-      });
+      taskDescriptions.push({ id: taskId, name: packet.node });
```

Comment:
No need for `input` here? Was a mistake before?

Reply:
Haven't seen usage of `input` in both JS nor Python, thus removed it in favour of matching Python